src/components/generics/date-range-picker/date-range-picker.tsx

```diff
@@ -50,7 +49,7 @@ const DateRangePicker: React.FC<Props> = ({
 
   const { locale } = useAppSelector(selectLocale);
 
-  const [ todayDate /* setTodayDate */ ] = React.useState(new Date());
+  const [ todayDate ] = React.useState(new Date());
```

Comment:
But does this actually need to be state if it is not getting updated at any point or could it just be a const?